package.json

```diff
@@ -44,15 +44,18 @@
     "renameschema": "scripts/rename-schema.sh",
     "presite": "npm run data && npm run build:site && npm run build:toc && npm run build:versions && scripts/create-example-pages",
     "site": "bundle exec jekyll serve --incremental",
+    "tsc:src": "tsc -b src/tsconfig.src.json",
+    "tsc:site": "tsc -b site/tsconfig.site.json",
     "prettierbase": "prettier '{src,test,test-runtime,site,typings}/**/*.{ts,md,css}'",
     "format": "tslint -p . --fix && npm run prettierbase -- --write",
     "lint": "tslint -p . && npm run prettierbase -- --check",
     "test": "jest test/ && npm run lint && npm run schema && jest examples/ && npm run test:runtime",
     "test:inspect": "node --inspect-brk ./node_modules/.bin/jest --runInBand test",
     "test:runtime": "TZ=America/Los_Angeles jest test-runtime/",
     "test:runtime:generate": "rm -Rf test-runtime/resources && VL_GENERATE_TESTS=true npm run test:runtime",
-    "watch:build": "npm run build:only && concurrently --kill-others -n Typescript,Rollup 'tsc -b src -w' 'rollup -c -w'",
-    "watch:site": "concurrently --kill-others -n Typescript,Rollup 'tsc -b site --watch' 'rollup -c site/rollup.config.js --watch'",
+    "watch:src": "npm run tsc:src -- -w",
```

Comment:
Why do we need both `watch:src` and `watch:build`?  

I'm a bit worried that we're adding too many commands that no one know how to use except you.

Reply:
I deleted it and updated the contribution docs.